pkg/bootstrapper/service.go

```diff
@@ -90,6 +117,28 @@ func (service *kubeletService) controlService(cmd svc.Cmd, desiredState svc.Stat
 	return nil
 }
 
+// controlDependentService sends a control signal to the kubelet dependent service
```

Comment:
`kubelet's dependent service`

Please call out that you are only going one level deep. i.e. you are not recursively checking the dependencies.